README.md

```diff
@@ -38,73 +39,132 @@ just install Ketrew:
-See `ketrew init --help` for more
-options, you can even ask it to generate self-signed TLS certificates.
-See also the [documentation](src/doc/The_Configuration_File.md)
-on the configuration file learn how to tweak it.
-
-You can check that the client or the server are configured:
-
-    ketrew print-configuration
-    ketrew print-configuration --configuration-profile server
-    ketrew print-configuration -P daemon
-
-You may then start a server:
-
-    KETREW_PROFILE=daemon ketrew start-server
-
-and then open the GUI:
-
-    ketrew gui
-
-which is just trying to load
-<http://127.0.0.1:8756/gui?token=my-not-so-secret-token> ☺
-
-You can always stop the server or check its status:
-
-    ketrew stop -P daemon
-    ketrew status -P daemon
+minimal workflow running.
+
+Before you can use Ketrew, you need to configure it:
+
+    $ ketrew init
+
+By default this will configure Ketrew in
+
+    $ ls $HOME/.ketrew/
+    authorized_tokens	configuration.ml
+
+You can check that the client or the server are configured (the client is returned by default):
+
+    $ ketrew print-configuration
+    [ketrew]
+        Mode: Client
+        Connection: "http://127.0.0.1:8756"
+        Auth-token: "755nRor8Q5z5nx7W22C6C078HF3YoY5PS29sEgNXxP4="
+        UI:
+            Colors: with colors
+            Get-key: uses `cbreak`
+            Explorer:
+                Default request: Targets younger than 1.5 days
+                Targets-per-page: 6
+                Targets-to-prefectch: 6
+        Misc:
+            Debug-level: 0
+            Plugins: None
+
+
+For the server:
+
+    $ ketrew pc server
+    [ketrew]
+    Mode: Server
+    Engine:
+        Database: "/Users/leonidrozenberg/.ketrew/database"
+        Unix-failure: does not turn into target failure
+        Host-timeout-upper-bound:
+        Maximum-successive-attempts: 10
+        Concurrent-automaton-steps: 4
+    UI:
+        Colors: with colors
+        Get-key: uses `cbreak`
+        Explorer:
+            Default request: Targets younger than 1.5 days
+            Targets-per-page: 6
+            Targets-to-prefectch: 6
+    HTTP-server:
+        Authorized tokens:
+            Inline (Name: 755nRor8Q5z5nx7W22C6C078HF3YoY5PS29sEgNXxP4=,
+            Value: "755nRor8Q5z5nx7W22C6C078HF3YoY5PS29sEgNXxP4=")
+            Path: "/Users/leonidrozenberg/.ketrew/authorized_tokens"
+        Daemonize: false
+        Command Pipe: Some "/Users/leonidrozenberg/.ketrew/command.pipe"
+        Log-path: Some "/Users/leonidrozenberg/.ketrew/server-log"
+        Return-error-messages: true
+        Max-blocking-time: 300.
+        Block-step-time: 3.
+        Listen: HTTP: 8756
+    Misc:
+        Debug-level: 0
+        Plugins: None
+
+Note that `pc` is a command _alias_ for `print-configuration`.
+Furthermore `daemon` is a shortcut for starting the `server` in
+[daemon](https://en.wikipedia.org/wiki/Daemon_%28computing%29) mode. You may
+now start a server:
+
+    $ ketrew start-server -P daemon
```

Comment:
I think that it is mentioned further down. Just not certain I want to get into discussing this right away.